client/packages/common/src/hooks/useUrlQuery/useUrlQueryParams.ts

```diff
@@ -49,21 +45,16 @@ export const useUrlQueryParams = ({
   initialSort,
   filters = [],
 }: UrlQueryParams = {}) => {
-  // do not coerce the filter parameter if the user enters a numeric value
-  // if this is parsed as numeric, the query param changes filter=0300 to filter=300
-  // which then does not match against codes, as the filter is usually a 'startsWith'
+  // Do not coerce the filter parameter if the user enters a numeric value
+  // If this is parsed as numeric, the query param changes filter=0300 to
+  // filter=300 which then does not match against codes, as the filter is
+  // usually a 'startsWith'
   const skipParse = filters.length > 0 ? filters.map(f => f.key) : ['filter'];
 
   const { urlQuery, updateQuery } = useUrlQuery({
     skipParse,
   });
 
-  const [storedRowsPerPage] = useLocalStorage(
-    '/pagination/rowsperpage',
-    DEFAULT_RECORDS_PER_PAGE
-  );
-  const rowsPerPage = storedRowsPerPage ?? DEFAULT_RECORDS_PER_PAGE;
-
```

Comment:
Removed -- don't think it'll affect current functionality, just means will always use 20 until using the new tables (don't think current one was working properly anyway)